### PR TITLE
UX cleanup of Dashboards

### PR DESCRIPTION
**Pull request by mindreeper2420**
This PR cleans up the Dashboards for Home and Space. Updates layouts for the cards with embedded lists, adds truncation for extra long Work Item titles and Space names and visual updates.

### CODE REVIEW

**Comment by fabric8cd** [comment]
@mindreeper2420 snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1129-fabric8-ui.int.rdu2c.fabric8.io

File: src/app/home/work-item-widget/work-item-widget.component.html [hunk 1]
```diff
@@ -1,44 +1,58 @@
 <div class="create-work-item-widget card-pf card-f8">
   <div class="card-pf-heading-bg">
     <div class="card-pf-heading">
-      <h2 class="card-pf-title">My work items</h2>
+      <h2 class="card-pf-title">My work items
+      <span *ngIf="spaces.length !== 0">
```
Comment (dlabrecq) [inline src/app/home/work-item-widget/work-item-widget.component.html]: This isn't what is shown in the mockup I worked from? In fact, the select menu covers the card title.

screen shot 2017-04-26 at 10 26 11 pm

Comment (mindreeper2420) [inline src/app/home/work-item-widget/work-item-widget.component.html]: @dlabrecq I worked with @catrobson on the update this evening - trying to save some spacing. I didn't have that overlap problem that you are showing here - what viewport resolution are you viewing this with?

Comment (mindreeper2420) [inline src/app/home/work-item-widget/work-item-widget.component.html]: @dlabrecq fix the overlap issue with PR https://github.com/fabric8io/fabric8-ui/pull/1130

Comment (mindreeper2420) [inline src/app/home/work-item-widget/work-item-widget.component.html]: @dlabrecq actually fixed here: https://github.com/fabric8io/fabric8-ui/pull/1144

**Pull request merged**